integration_tests/ethereum/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:14-alpine3.13
+
+RUN apk update
+RUN apk add --no-cache git python3 make build-base
+
+COPY package.json package.json
+COPY yarn.lock yarn.lock
+
+RUN yarn install --production=false
+RUN npm config set user 0
+
+COPY . .
+
+ENV ARCHIVE_NODE_URL=""
```

Comment:
I could leave a comment? Since the `integration_tests` dir is replicated across three repos right now maybe we should move it to a single test repo and document it there?